slashing/program/src/shred.rs

```diff
@@ -333,20 +332,17 @@ impl Shred {
         {
             return false;
         }
-        fn get_payload(shred: &Shred) -> &[u8] {
+        fn get_payload<'a>(shred: &'a Shred<'a>) -> &'a [u8] {
```

Comment:
nit: does the shred reference need to have `'a` lifetime? Since the payload is coming from inside the shred, I'm thinking we can have `(shred: &'b Shred<'a>) -> &'a [u8]`. Is that correct?

Reply:
No it doesn't you're absolutely right 9174be41bec85352c21555fcf671daff13e3c249